OsmAnd/src/net/osmand/plus/activities/FavoritesTreeFragment.java

```diff
@@ -943,6 +974,27 @@ public void onClick(View v) {
 			name.setTypeface(Typeface.DEFAULT, visible ? Typeface.NORMAL : Typeface.ITALIC);
 			name.setTextColor(getResources().getColor(visible ? enabledColor : disabledColor));
 			distanceText.setText(distance);
+			if (groupPosition == 0) {
+				String address = model.getDescription() != null ? "• " + model.getDescription() : "";
+				distanceText.setText(distance + address);
+				switch (model.getName()) {
+					case FavouritesDbHelper.HOME:
+						icon.setImageDrawable(getMyApplication().getUIUtilities().getThemedIcon(R.drawable.ic_action_home_dark));
```

Comment:
We have favorites in other parts of the app. Will it be copy-paste? BTW - looks like other places are not handled at all. For example - Search.